httplib/httplib.go

```diff
@@ -512,11 +512,13 @@ func (b *BeegoHTTPRequest) DoRequest() (resp *http.Response, err error) {
 	// retries default value is 0, it will run once.
 	// retries equal to -1, it will run forever until success
 	// retries is setted, it will retries fixed times.
+	// Sleeps for a 400ms inbetween calls to reduce spam
 	for i := 0; b.setting.Retries == -1 || i <= b.setting.Retries; i++ {
 		resp, err = client.Do(b.req)
 		if err == nil {
 			break
 		}
+		time.Sleep(400 * time.Millisecond)
```

Comment:
I think making this configurable would be better.

Reply:
Sure, do you think a fixed value set by the user would be better or a multiplier i.e 2x the last wait time like the TCP slow start algorithm

Reply:
If you can define an interfaces that allow users to implement their strategy, it will be more scalable. But now, I think a fixed value looks good. What do you think?

Reply:
Yeah a fixed value would do the job. It would only have to trigger this on edge cases where its internet connection so it shouldn't be too much of an important change.